users/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http              import JsonResponse
+from django.views             import View
+from core.utils               import *
+
+from django.core.exceptions   import ValidationError 
+from wesop.settings           import SECRET_KEY,ALGORITHM
+from core.validator import *
+
+from users.models import *
```

Comment:
import 할 때 package의 성격에 따라 구분을 지어 줍니다.
1. 내장 package
2. pip 설치한 외부 package
3. 프로젝트 코드 내부에 있는 package
4. 그리고 *는 사용을 자제해 주세요.

```suggestion
import json

import bcrypt
import jwt
from django.http            import JsonResponse
from django.views           import View
from django.core.exceptions import ValidationError 

from core.utils     import 
from core.validator import 
from wesop.settings import SECRET_KEY,ALGORITHM
from users.models   import 
```